code/datums/status_effects/debuffs.dm

```diff
@@ -9,11 +9,12 @@
 	alert_type = null
 
 /datum/status_effect/incapacitating/on_creation(mob/living/new_owner, set_duration)
+	if(new_owner.status_flags & GODMODE)
+		return qdel(src)
```

Comment:
perhaps instead add it to the parent? and delete all status_effects on godmode

Reply:
I put it at incapacitating because not all status effects are negative. 

I thought about purging status effects on godmode but decided against it because you recover from them normally in godmode anyway, and where it's actually used outside of admemery (king petrify) would look weird if people recover from stuns by getting turned to stone.